packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -38,11 +39,18 @@ export type ChatScreenProps = {
   onRenderMessage?: (messageProps: MessageProps, defaultOnRender?: MessageRenderer) => JSX.Element;
   onRenderTypingIndicator?: (typingUsers: CommunicationParticipant[]) => JSX.Element;
   onFetchParticipantMenuItems?: ParticipantMenuItemsCallback;
+  styles?: MessageThreadStylesProps;
```

Comment:
This implies styles for the whole chat screen but only has styles for the MessageThread. Open to renaming this to messageThreadStyles or having something like:
```
interface ChatScreenStyles {
   messageThreadStyles: MessageThreadStylesProps
}

...

export type ChatScreenProps = {
   ...
   styles: ChatScreenStyles;
}

```

Reply:
yes lets change that to align with Prathmesh's latest change I will get in on that.